docs/tutorial/t_reconstruction_system/dense_slam.rst

```diff
@@ -13,17 +13,17 @@ In a SLAM system, we maintain a ``model`` built upon a :ref:`voxel_block_grid`,
 
 .. literalinclude:: ../../../examples/python/t_reconstruction_system/dense_slam.py
    :language: python
-   :lineno-start: 45
-   :lines: 27,46-54
+   :lineno-start: 26
+   :lines: 8,27-35
```

Comment:
File integrate_custom, dense_slam and ray_casting.rst have quite a bit of indent. 

One way to remove indent is to add dedent option but that one will also dedent the line 8 which has path to file  in comment `# examples/python/t_reconstruction_system/dense_slam.py`.

I can remove line number 8 and add prepend option with filename, but that one only works with sphinx version 5.0 - examples/python/t_reconstruction_system/dense_slam.py

The upgrade was simple but it caused a lot of warnings like - 

```
docstring of open3d.cpu.pybind.visualization.gui.Widget.EventCallbackResult:1: WARNING: duplicate object description of open3d.cpu.pybind.visualization.gui.Widget.EventCallbackResult, other instance in python_api/open3d.visualization.gui.TreeView, use :noindex: for one of them
```

So, for now, I am leaving this indentation issue unresolved.

Reply:
I think the indentation is OK - it does not distract from the example code / make it hard to read.